src/poolman.cpp

```diff
@@ -0,0 +1,96 @@
+
```

Comment:
Nit: Missing license, missing header group ordering and also std namespace polution.